Build/Build.cs

```diff
@@ -145,9 +145,9 @@ class Build : NukeBuild
                 .SetResultsDirectory(TestResultsDirectory)
                 .CombineWith(cc => cc
                     .SetProjectFile(project)
-                    .AddLoggers($"trx;LogFileName={project.Name}.trx")));
-            
-            ReportTestOutcome(globFilter: $"*{project.Name}.trx");
+                    .AddLoggers($"trx;LogFileName={project.Name}.trx")), completeOnFailure: true);
```

Comment:
Had enabled this, because when the tests continue, it gives a better overview of how many tests failed in the summary.

Only drawback is, that we don't have a "fast-failure" anymore 🤔